### ISSUE TITLE
Problem with uneven-indent rule

### ISSUE THREAD

**Issue by mnojek** [MarketSquare/robotframework-robocop#375]
Consider following code snippet:
```
*** Settings ***
Documentation  Suite doc


*** Keywords ***
My Keyword    # robocop: disable
    [Documentation]  Some doc
    No Operation

```
It does not report any issues.
But if we change line 6th and decrease the number of spaces between the keyword name and the comment (which is a Robocop disabler) by 2, so the line looks like this:
```
My Keyword  # robocop: disable
```
then we have this issue reported by Robocop:
```indent_robocop.robot:7:5 [W] 1007 Line is over-indented (uneven-indent)```
which should not happen.

**Comment by bhirsz** [comment]
I can confirm, it also happen with any other comment:
![image](https://user-images.githubusercontent.com/8532066/123086614-e8d2b900-d423-11eb-9afd-397d873ec49c.png)

**Comment by bhirsz** [comment]
It also don't detect obvious rule violations such as:
```
Test 3
     [Tags]    could-be-forced
    ${ret}    My Python Keyword    abc
```

**Comment by matusaurio** [comment]
Hi, I'm also facing an issue with rule 1007

When I need to exclude some rule to be applied in a code fragment, I do the following and I get the error
`Line is under-indented (uneven-indent)`

```
Keyword 1
    Some code
# robocop: disable
    Some code I don't want to be checked by robocop
# robocop: enable

Keyword 2
    Log  2
```

But if I add the required indentation to the `# robocop: disable` all the rules are applied over my code, so the disable rule is ignored, and that is not the behaviour I'm looking for.

**Comment by bhirsz** [comment]
Thanks for the report. By the looks of it, any rule should ignore comments with robocop disabler (which is not happening for you). The syntax you're using is for block disablers (you can disable one or more lines) and it's required that is start from first char in line. If you only want to disable one line you can for now disable only this line:
```
Keyword 1
    Some code
    Some code I don't want to be checked by robocop # robocop: disable

Keyword 2
    Log  2
```
And issue mentioned by you will be fixed.

**Comment by matusaurio** [comment]
That's the thing, I want to exclude a block of code (sorry if my example wasn't clear enough, I edited it), that's why I use the block disabler and as explain above, it'll not work if it's indented which at the same time clashes with the rule 1007.
